index.html

```diff
@@ -47,3 +47,35 @@ <h2>Update Exchange Rate</h2>
     <script src="forex.js"></script>
 </body>
 </html>
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
You are creating a new `html` document here. There should be only one per file. Either create a new file, or modify the existing code. I recommend the latter so you keep growing your currency converter project.